plugins/explore/src/components/DomainExplorerContent/DomainExplorerContent.tsx

```diff
@@ -32,15 +33,19 @@ import {
 
 import { useApi } from '@backstage/core-plugin-api';
 
-const Body = () => {
+type BodyProps = PropsWithChildren<{
+  filter: Record<string, string | symbol | (string | symbol)[]>;
+}>;
+const Body = (props: BodyProps) => {
+  const { filter } = props;
   const catalogApi = useApi(catalogApiRef);
   const {
     value: entities,
     loading,
     error,
   } = useAsync(async () => {
     const response = await catalogApi.getEntities({
-      filter: { kind: 'domain' },
+      filter: filter,
     });
     return response.items as DomainEntity[];
```

Comment:
Gotta say this is a bit tricky, since we're assuming that we're dealing with `Domain` entities here. Would it perhaps make sense to explore some other higher form of abstraction that lets us achieve the same thing? Some kind of entity card grid (also to be used in the scaffolder? 😱)

Also wondering whether this might be one of those cases where it's better to implement something custom in your own project too, using tools like `ItemCardGrid` and `EntityListProvider`

Reply:
Yeah I am currently using something custom! So, if that's the way to go then I am fine with leaving this alone 😄  
We can also explore the entity card gird, but that would make sense to do in a different PR. So, I am closing this PR, thanks for the feedback! 😸